vanilla-framework/vanilla-framework
# Add ./run build scripts

@nottrobin When trying to run `./run jekyll` I get an error: `Command not recognised.` - any ideas?

## Done

Added run scripts

## QA

Check each command runs without error:

```
./run help
./run develop
./run jekyll
./run test
./run build
```

## Details

Fixes #978
